### PR TITLE
Add Payback Loyalty Program

### PR DESCRIPTION
**Pull request by kmpoppe**

### CODE REVIEW

**Comment by kmpoppe** [comment]
Alexa rank 11,748 as per 22/01/22

File: img/p/payback.de.svg [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                <path d="M65.1005539,8.73779947 L62.2338638,3.364677 L64.8079431,0.43256433 L62.157576,0.43256433 L61.4078355,1.27499914 C60.823211,1.92990063 60.0419701,2.84837434 59.6393316,3.39925345 C59.6270898,3.41522309 59.6091748,3.43778569 59.5985751,3.45258324 C59.6229096,2.84837434 59.6364951,2.10424801 59.6364951,1.40803058 L59.6364951,0.43256433 L57.2197688,0.43256433 L57.2197688,8.73779947 L59.6364951,8.73779947 L59.6364951,6.23422972 L60.5426928,5.26535644 L60.5994235,5.20411509 L62.5603637,8.73779947 L65.1005539,8.73779947" id="Fill-21"></path>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (Carlgo11) [inline img/p/payback.de.svg]: It's amazing what vector graphics can do!
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" style="fill:none;background:#0046aaff;stroke-width:1.5;stroke:#fff" viewBox="0 0 115.5 115.5"><g id="b"><circle id="a" cx="21.5" cy="21.5" r="12.9"/><use x="35.7" href="#a"/><use x="71.3" fill="#fff" href="#a"/></g><use y="35.7" href="#b"/><use x="35.7" y="35.7" fill="#fff" href="#a"/><use y="71.3" href="#b"/></svg>
```

File: entries/p/payback.de.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "Payback": {
+    "domain": "payback.de",
+    "url": "https://www.payback.de/",
```
Comment (Carlgo11) [inline entries/p/payback.de.json]: The url tag is no longer needed.
```suggestion
```

File: entries/p/payback.de.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "tfa": [
+      "totp",
+      "custom-software"
```
Comment (Carlgo11) [inline entries/p/payback.de.json]: Is it true TOTP RFC-6238 or does it only work via the PAYBACK app?

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/p/payback.de.json (hunk 3 above)
Comment (kmpoppe) [reply entries/p/payback.de.json]: Nope, just custom software, no "real TOTP"

**Comment by kmpoppe** [comment]
Obviously egnyte wasn't such a good example to start from ;-)

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
The the build warning says:
```
 A `custom-software` tag is needed since it has been included in the `tfa` array.
```

**Comment by kmpoppe** [comment]
Had to find the time to find a matching example :)

**Review by Carlgo11 — APPROVED** [review]
Danke!

**Review by phallobst — APPROVED** [review]

**Pull request merged**